xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -715,8 +740,8 @@ Draining occurs when a request is cancelled or a response is disposed prior to f
         <ReturnType>System.Net.Http.HeaderEncodingSelector&lt;System.Net.Http.HttpRequestMessage&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets a callback that encodes response headers values.</summary>
+        <value>The header encoding selector callback to decode the value for the specified response header name, or <see langword="null"/> indicating the default behavior.</value>
```

Comment:
Here too

Reply:
it should be `decode`. @MihaZupan can you confirm?

Reply:
```suggestion
        <value>The header encoding selector callback to decode the value for the specified response header name, or <see langword="null"/> to indicate the default behavior.</value>
```